### PR TITLE
Simple Interest Calculator 

### PR DESCRIPTION
**Pull request by Mo7amedSMahdi**
- Add gitIgnore
- Setup linters
- Add index.html
- Add script.js
- Add Style.css

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+                    <option value="2">2</option>
+                    <option value="3">3</option>
+                    <option value="4">4</option>
+                    <option value="5">5</option>
+                    <option value="6">6</option>
+                    <option value="7">7</option>
+                    <option value="8">8</option>
+                    <option value="9">9</option>
+                    <option value="10">10</option>
+
```
Comment (Mo7amedSMahdi) [inline index.html]: - [ ] Kindly remove unnecessary extra spaces. Removes Extra spaces make the code looks cleaner.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+                    <option value="9">9</option>
+                    <option value="10">10</option>
+    
+                </select>
+            </div>
+            <div class="form-input">
+                <button onclick="compute()">Compute Interest</button>
+            </div>
+        </div>
+        <br /><br />
```
Comment (Mo7amedSMahdi) [inline index.html:44]: - [ ] Kindly use padding and margin. This is more semantic to use margin and padding. Chekout [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=Use%20the%20appropriate%20tags%20for%20each%20element%20(e.g.%2C%20links%2C%20titles%2C%20etc.)%20and%20use%20HTML5%20semantic%20tags%20(e.g.%2C%20header%2C%20nav)%20over%20divs.)

File: index.html [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+            &#169; Everyone Can Get Rich
+
+
+        </footer>
+    </div>
+
+</body>
+
+</html>
```
Comment (Mo7amedSMahdi) [inline index.html]: Kindly remove unnecessary extra spaces. Removes Extra spaces make the code looks cleaner.

**Review by Mo7amedSMahdi — COMMENTED** [review]
# Required Changes ♻️ 
_Check the comments under the review._

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

- [ ] Please fix linter errors ❌ 



Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi **@Mo7amed,**

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!

## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

-  ***You can use this below as a hint to improve on your second review***

- [ ] Great job on adding linters to your project👏 . However, they are not passing yet... (now add a few words explaining to the student how linters are fair in a project)
- [ ] You most of the time talked about empty spaces, please tag HTML best practices too 😄 
- [ ] Kindly, Add a descriptive README file to your project.... (Now add a few words here too explaining to the student how the readme is important in a project)

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏 💻  

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Mo7amedSMahdi) [inline index.html]: - [ ]  Kindly remove unnecessary extra spaces. Removes Extra spaces make the code looks cleaner.
checkout [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

File: index.html [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+        <span id="result">
+
+        </span>
+        <br /><br />
+        <footer>
+
+            &#169; Everyone Can Get Rich
+
+
+        </footer>
```
Comment (Mo7amedSMahdi) [inline index.html:54]: - [ ]  Kindly remove unnecessary extra spaces. Removes Extra spaces make the code looks cleaner.
checkout [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

File: index.html (hunk 3 above)
Comment (Mo7amedSMahdi) [inline index.html]: - [ ]  Kindly remove unnecessary extra spaces. Removes Extra spaces make the code looks cleaner.
checkout [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

**Review by Mo7amedSMahdi — COMMENTED** [review]
### Hi **@Mo7amed,**

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!

## STATUS: CHANGES REQUESTED  ♻️ 
# Required Changes ♻️ 
_Check the comments under the review._

 [ ] Great job on adding linters to your project👏 . However, they are not passing yet.
- [ ] Kindly, Add a descriptive README file to your project ( Readme is very important to understand your project and how to use it).



Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Kingstalux) [reply index.html]: Excellent.

**Review by Kingstalux — APPROVED** [review]
Hi @Mo7amedSMahdi , 
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

Good job implementing the required changes 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

- [ ] (OPTIONAL)  It would be great if you could add emojis to your comments to make them more friendly 😄 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Mo7amedSMahdi** [comment]
Thanks for mentioning that I will add more emojis to my comment next time.

**Pull request merged**